### ISSUE TITLE
zone: defaults to master but i do not want to configure zone at all.

### ISSUE THREAD

**Issue by NeverUsedID** [T-Systems-MMS/ansible-collection-icinga-director#25]
In our environment the zone is defined in the host template:

```
imports: 
  - WINDOWS SERVER AGENT
  - {{ icinga.satelliteTemplate }}
```

This will not work, as zone: defaults to master and the zone will configured for each created host. I would like to leave zone: empty.  Is there any way?

**Comment by rndmh3ro** [comment]
Defaulting the zone to `master` is wrong. There's no requirement for this and it should be by default empty. I'll fix this.

**Comment by rndmh3ro** [comment]
This should now be fixed in version 1.2.2 found in the galaxy: https://galaxy.ansible.com/t_systems_mms/icinga_director
